synapse/tools/cryo/cat.py

```diff
@@ -55,11 +59,36 @@ def main(argv, outp=s_output.stdout):
 
         return 0
 
-    for item in cryo.slice(path, opts.offset, opts.size, opts.timeout):
-        if opts.jsonl:
-            outp.printf(json.dumps(item[1], sort_keys=True))
+    def except_wrap(it, error_str_func):
```

Comment:
lets move this guy up above main() since he's not using any locals

Reply:
why?

Reply:
ah, yeah he used to. ok